custom/icds_reports/ucr/tests/test_child_health_monthly.py

```diff
@@ -1124,3 +1237,73 @@ def test_ebf_no_ebf_reasons(self):
              ),
         ]
         self._run_iterative_monthly_test(case_id=case_id, cases=cases)
+
+    def test_cf(self):
+        case_id = uuid.uuid4().hex
+        case_id_2 = uuid.uuid4().hex
+        self._create_case(
+            case_id=case_id,
+            dob=date(2015, 5, 12),
+            date_opened=datetime(2015, 5, 14),
+            date_modified=datetime(2016, 3, 12),
+        )
+        self._create_case(
+            case_id=case_id_2,
+            dob=date(2015, 05, 12),

```

Comment:
github syntax highlighting caught this - `05` is an ocatal 5 in python, which happily is the same as a decimal 5, though probably better to just have `5`